src/static/js/vendors/farbtastic.js

```diff
@@ -33,7 +33,7 @@ $._farbtastic = function (container, options) {
   fb.linkTo = function (callback) {
     // Unbind previous nodes
     if (typeof fb.callback == 'object') {
-      $(fb.callback).unbind('keyup', fb.updateValue);
+      $(fb.callback).off('keyup', fb.updateValue);
```

Comment:
I prefer to minimize diff to the upstream version so that newer versions can be merged in with fewer conflicts.